sdk/quota/arm-quota/src/azureQuotaExtensionAPI.ts

```diff
@@ -1,202 +1,106 @@
+import { _getGroupQuotaLocationSettingsOperations } from "./classic/groupQuotaLocationSettings/index.js";
+import type { GroupQuotaSubscriptionAllocationOperations } from "./classic/groupQuotaSubscriptionAllocation/index.js";
+import { _getGroupQuotaSubscriptionAllocationOperations } from "./classic/groupQuotaSubscriptionAllocation/index.js";
+import type { GroupQuotaSubscriptionAllocationRequestOperations } from "./classic/groupQuotaSubscriptionAllocationRequest/index.js";
+import { _getGroupQuotaSubscriptionAllocationRequestOperations } from "./classic/groupQuotaSubscriptionAllocationRequest/index.js";
+import type { GroupQuotaSubscriptionRequestsOperations } from "./classic/groupQuotaSubscriptionRequests/index.js";
+import { _getGroupQuotaSubscriptionRequestsOperations } from "./classic/groupQuotaSubscriptionRequests/index.js";
+import type { GroupQuotaSubscriptionsOperations } from "./classic/groupQuotaSubscriptions/index.js";
+import { _getGroupQuotaSubscriptionsOperations } from "./classic/groupQuotaSubscriptions/index.js";
+import type { GroupQuotaUsagesOperations } from "./classic/groupQuotaUsages/index.js";
+import { _getGroupQuotaUsagesOperations } from "./classic/groupQuotaUsages/index.js";
+import type { GroupQuotasOperations } from "./classic/groupQuotas/index.js";
+import { _getGroupQuotasOperations } from "./classic/groupQuotas/index.js";
+import type { QuotaOperations } from "./classic/quota/index.js";
+import { _getQuotaOperations } from "./classic/quota/index.js";
+import type { QuotaOperationOperations } from "./classic/quotaOperation/index.js";
+import { _getQuotaOperationOperations } from "./classic/quotaOperation/index.js";
+import type { QuotaRequestStatusOperations } from "./classic/quotaRequestStatus/index.js";
+import { _getQuotaRequestStatusOperations } from "./classic/quotaRequestStatus/index.js";
+import type { UsagesOperations } from "./classic/usages/index.js";
+import { _getUsagesOperations } from "./classic/usages/index.js";
+import type { TokenCredential } from "@azure/core-auth";
+import type { Pipeline } from "@azure/core-rest-pipeline";
 
-export class AzureQuotaExtensionAPI extends coreClient.ServiceClient {
-  $host: string;
-  apiVersion: string;
-  subscriptionId?: string;
+export { AzureQuotaExtensionAPIOptionalParams } from "./api/azureQuotaExtensionAPIContext.js";
 
-  /**
-   * Initializes a new instance of the AzureQuotaExtensionAPI class.
-   * @param credentials Subscription credentials which uniquely identify client subscription.
-   * @param subscriptionId The ID of the target subscription. The value must be an UUID.
-   * @param options The parameter options
-   */
+export class AzureQuotaExtensionAPI {
+  private _client: AzureQuotaExtensionAPIContext;
+  /** The pipeline used by this client to make requests */
+  public readonly pipeline: Pipeline;
+
+  /** Microsoft Azure Quota Resource Provider */
   constructor(
```

Comment:
constructor has an overload, please manually update this. there is an issue in codegen to track this.

Reply:
fixed, could you help review the change?